isaric/parsers/isaric-core.toml

```diff
@@ -0,0 +1,2546 @@
+  ref = "Y/N/NK"
+
+[subject.has_asthma]
+  field = "asthma_mhyn"
+  description = "Clinician-diagnosed asthma"
+  ref = "Y/N/NK"
+
+[subject.has_chronic_cardiac_disease]
+  combinedType = "any"
+
+  [[subject.has_chronic_cardiac_disease.fields]]
+    field = "chroniccard_mhyn"
+    description = "<acronym title=\"Include any of coronary artery disease, heart failure, congenital heart disease, cardiomyopathy, rheumatic heart disease.\">Chronic cardiac disease <i>(not hypertension)</i></acronym>"
+    ref = "Y/N/NK"
+
+  [[subject.has_chronic_cardiac_disease.fields]]
+    field = "chronicpul_mhyn"
+    description = "<acronym title=\"Include any of chronic obstructive pulmonary disease (chronic bronchitis, chronic obstructive pulmonary disease (COPD), emphysema), cystic fibrosis, bronchiectasis, interstitial lung disease, pre-existing requirement for long term oxygen therapy. Do not include asthma.\">Chronic pulmonary disease <i>(not asthma)</i></acronym>"
+    ref = "Y/N/NK"
+
+[subject.has_chronic_kidney_disease]
+  field = "renal_mhyn"
+  description = "<acronym title=\"Please include any of clinician-diagnosed chronic kidney disease, chronic estimated glomerular filtration rate < 60mL/min/1.73m2, history of kidney transplantation\">Chronic kidney disease</acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_dementia]
+  field = "dementia_mhyn"
+  description = "<acronym title=\"This is defined a clinical diagnosis of dementia\">Dementia</acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_diabetes]
+  combinedType = "any"
+
+  [[subject.has_diabetes.fields]]
+    field = "diabetes_mhyn"
+    description = "4b.13. Diabetes without complication"
+    ref = "Y/N/NK"
+
+  [[subject.has_diabetes.fields]]
+    field = "diabetiscomp_mhyn"
+    description = "4b.12. Diabetes with complication"
+    ref = "Y/N/NK"
+
+  [[subject.has_diabetes.fields]]
+    field = "diabetes_mhyn_2"
+    description = "<acronym title=\"Type 1 or Type 2 diabetes mellitus requiring oral or subcutaneous treatment. Please indicate whether type 1 or type 2.\">Diabetes Mellitus</acronym>"
+
+    [subject.has_diabetes.fields.values]
+      1 = "yes - type 1"
+      4 = "yes - type 2"
+      5 = "yes - gestational"
+      2 = false
+
+  [[subject.has_diabetes.fields]]
+    field = "hba1c_mborres"
+    description = "<i>Diabetes Mellitus</i> - if YES, HbA1C result (within last 6 months)"
+
+[subject.has_hiv]
+  combinedType = "any"
+
+  [[subject.has_hiv.fields]]
+    field = "aidshiv_mhyn"
+    description = "4b.10. AIDS/HIV"
+    ref = "Y/N/NK"
+
+  [[subject.has_hiv.fields]]
+    field = "aidshiv_mhyn_2"
+    description = "<acronym title=\"History of laboratory-confirmed HIV infection. Indicate whether or not the patient is on ART (antiretroviral therapy)\">AIDS / HIV</acronym>"
+
+    [subject.has_hiv.fields.values]
+      1 = "yes - on art"
+      4 = "yes - not on art"
+      2 = false
+
+  [[subject.has_hiv.fields]]
+    field = "aidshiv_cd4"
```

Comment:
Should 'has_hiv' be boolean? or require a boolean output? ... I suppose CD4 count be translated to boolean - not sure how. Or CD4 count should be an observation / measurement?

Reply:
I'm not sure where this is... it's not in the most recent file anyway :) But yes, has_hiv should be boolean.